templates/settings.html

```diff
@@ -6,30 +6,37 @@
 
 {% block main %}
 <div class="container">
-    <div>
-        <h4>Current Budget</h4>
-        <p>{{ daily_budget | usd }}</p>
-    </div>
     <div>
         <h4>Preferences</h4>
-        <ul>
+        <!-- <ul>
             {% for pref in current_user_prefs %}
                 <li> {{ pref }} </li>
             {% endfor %}
-        </ul>
+        </ul> -->
         <form action="/settings" method="post">
             <div>
+                {% for cuisine in all_cuisine_types %}
+                    <div class="form-check form-switch">
+                        {% if cuisine.cuisine_type in current_user_prefs %}
+                            <input class="form-check-input" type="checkbox" id="{{ cuisine.cuisine_type }}" name="cuisine_selections" value="{{ cuisine.cuisine_type }}" checked>
+                        {% else %}
+                            <input class="form-check-input" type="checkbox" id="{{ cuisine.cuisine_type }}" name="cuisine_selections" value="{{ cuisine.cuisine_type }}">
+                        {% endif %}
+                        <label class="form-check-label" for="{{ cuisine.cuisine_type }}">{{ cuisine.cuisine_type }}</label>  
+                    </div>
+                {% endfor %}
+            </div>
+            <div>
+                <h4>Current Budget</h4>
+                <p>{{ daily_budget | usd }}</p>
+            </div>
+            <div>
+                <input name="new_budget" autocomplete="off" id="new_budget" type="number" placeholder="$0.00" step=".01">
                 <div>
```

Comment:
This might not work exactly because I don't have a test bed but it just seems cleaner to have the input the same as the display

```suggestion
                <h4>Budget</h4>
                <input name="new_budget" autocomplete="off" id="new_budget" type="number" placeholder="{{ daily_budget | usd if daily_budget is not none else '0.00' }}" step=".01">
            </div>
```

Reply:
Potentially even shortened to:

```python
<input name="new_budget" autocomplete="off" id="new_budget" type="number" placeholder="{{ daily_budget | usd if daily_budget else '0.00' }}" step=".01">
```